bake/bake_test.go

```diff
@@ -555,7 +555,7 @@ services:
 	require.Equal(t, "Dockerfile", *m["app1"].Dockerfile)
 	require.Equal(t, ".", *m["app1"].Context)
 	require.Equal(t, "Dockerfile", *m["app2"].Dockerfile)
-	require.Equal(t, ".", *m["app2"].Context)
+	require.Equal(t, "/src/bake", *m["app2"].Context)
```

Comment:
It fixes the test but don't think it's right. We are not matching the HCL/JSON implementation anymore :confused:.